storybook/main.cpp

```diff
@@ -58,7 +58,11 @@ int main(int argc, char *argv[])
     qmlRegisterSingletonType<CacheCleaner>(
                 "Storybook", 1, 0, "CacheCleaner", cleanerFactory);
 
+#ifdef Q_OS_WIN
+    const QUrl url(QUrl::fromLocalFile(QML_IMPORT_ROOT + QStringLiteral("/main.qml")));
```

Comment:
Won't this also work for other platforms to simplify it and drop the compile time condition?

Reply:
At this moment I'm not sure, because it would also affect MacOS devs.
This line doesn't change and I'm 100% sure that will cover all cases.
I think it's not worth investigating it.